client/android/app/build.gradle

```diff
@@ -25,7 +25,7 @@ if (flutterVersionName == null) {
 android {
     namespace "com.appveyor.flet"
     compileSdkVersion flutter.compileSdkVersion
-    ndkVersion flutter.ndkVersion
+    ndkVersion "25.1.8937393"
```

Comment:
Accidental check-in or it's required by a plugin?

Reply:
Could you point me to the docs for that please?

Reply:
i couldn't find the docs for it 😅, but when i tried to run the `flutter build apk` command without `ndkVersion "25.1.8937393"`, it is giving an error saying 

```
One or more plugins require a higher Android NDK version.
Fix this issue by adding the following to C:\Users\user\OneDrive\Desktop\projects\flet\client\android\app\build.gradle:
android {
  ndkVersion "25.1.8937393"
  ...
}
```